internal/pkg/mount/mount.go

```diff
@@ -266,6 +268,9 @@ func (p *Point) Mount() (err error) {
 // Unmount attempts to retry an unmount on EBUSY. It will attempt a
 // retry every 100 milliseconds over the course of 5 seconds.
 func (p *Point) Unmount() (err error) {
+	p.Lock()
+	defer p.Unlock()
```

Comment:
There are much more places where Point's fields are accessed and should be protected by mutex.

Or maybe we can say that Point is not thread-safe, and it is caller's responsibility to synchronize access to Point's methods

Reply:
I can move it to the `system.go` then.